modules/web-console/frontend/app/components/list-of-registered-users/categories.ts

```diff
@@ -21,10 +21,10 @@ export const categoriesFn = ($translate: ng.translate.ITranslateService) => [
     {name: 'Activated', visible: false, enableHiding: true},
     {name: 'Company', visible: true, enableHiding: true},
     {name: 'Country', visible: true, enableHiding: true},
-    {name: 'Last login', visible: false, enableHiding: true},
+    {name: 'Last login', visible: true, enableHiding: true},
     {name: 'Last activity', visible: true, enableHiding: true},
-    {name: 'Configurations', visible: false, enableHiding: true},
-    {name: 'Total activities', visible: true, enableHiding: true},
+    {name: 'Configurations', visible: true, enableHiding: true},
+    {name: 'Total activities', visible: false, enableHiding: true},
```

Comment:
Why you change this?

Reply:
Total activities are hidden because they are not implemented in the Java backend.
Configuration columns are shown because they are fixed and show useful data.

Reply:
We need `Configurations` columns because it helps us to find user with 4000+ caches.
And it is annoying to make them visible by hands.